### DISCUSSION TITLE
React Frontend/Backend Management

### DISCUSSION THREAD

**Discussion by Caleb-Cohen** [Together-100Devs/Together#25] [General]
Since installing the login with the discord button, we've created a frontend folder to allow this functionality. There have been some concerns about how to properly run that in the development environment and organization of the code, as now there are multiple package.json, .gitignore, and Readme files. 

I want to start a discussion on how to improve the organization of the file layout and make the application more accessible so users with less react experience can continue to work on non-react tasks.

**Comment by Caleb-Cohen** [comment]
I had a conversation today with GC. He explained to me that the double terminal setup is not ideal, but it's the simplest way to get things running. Still open to suggestions, but this is probably the way we'll keep it.

**Comment by ghost** [comment]
Posing the question weather or not this topic is outdated or might become irrelevant; reffering to people having trouble running, assuming we shift over to the Next project, I feel confident that could be refactored in a few hours maybe more if a freak accident happens.
